<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestiongroupe</class>
 <widget class="QMainWindow" name="gestiongroupe">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>823</width>
    <height>607</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget_8">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>271</width>
      <height>119</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout_8">
     <item row="1" column="0">
      <widget class="QLabel" name="label_48">
       <property name="text">
        <string>Nom du groupe</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_49">
       <property name="text">
        <string>Prix</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_50">
       <property name="text">
        <string>Description</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="groupeNomEdit">
       <property name="maxLength">
        <number>15</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="groupeDescritpionEdit">
       <property name="maxLength">
        <number>50</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_58">
       <property name="text">
        <string>Type de camp</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="groupeTypeCampEdit">
       <item>
        <property name="text">
         <string>Aucun</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Patinage artistique</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hockey</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Powerskating</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QDoubleSpinBox" name="groupePrixSpinBox">
       <property name="locale">
        <locale language="English" country="Canada"/>
       </property>
       <property name="maximum">
        <double>500.000000000000000</double>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_59">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Nouveau groupe</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_47">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>200</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Recherche de groupe</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_30">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>240</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_11">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Rechercher</string>
    </property>
   </widget>
   <widget class="QTableView" name="rechercheGroupetableView">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>701</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ajouterGroupeButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Valider</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>823</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
